docs/source/release_notes.rst

```diff
@@ -4,6 +4,192 @@
+- `[#616] [FEniCS] Ensure that amax and dofs behave correctly in the MPI discributed case <https://github.com/pymor/pymor/pull/616>`_
+
+Improved greedy algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#757] Refactor greedy algorithms <https://github.com/pymor/pymor/pull/757>`_
+
+
+Numerical linear algebra algorithms
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#665] adding randomized algorithms <https://github.com/pymor/pymor/pull/665>`_
+- `[#817] SVD for vectorarrays using Gram-Schmidt <https://github.com/pymor/pymor/pull/817>`_
+- `[#577] Gram-Schmidt with R <https://github.com/pymor/pymor/pull/577>`_
+
+
+Support for low-rank operators
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#743] Low-rank updated operators and Sherman-Morrison-Woodbury formula <https://github.com/pymor/pymor/pull/743>`_
+
+
+Improved string representations of pyMOR objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#652] __repr__ and __str__ for models <https://github.com/pymor/pymor/pull/652>`_
+- `[#706] Add generic __repr__ method to BasicInterface <https://github.com/pymor/pymor/pull/706>`_
+
+
+Easier working with immutable objects
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#732] Add __auto_init method and use it where applicable <https://github.com/pymor/pymor/pull/732>`_
+- `[#705] Add new_type argument to with_ <https://github.com/pymor/pymor/pull/705>`_
+- `[#694] Ensure that with_ works for all immutable objects <https://github.com/pymor/pymor/pull/694>`_
+
+
+project and assemble_lincomb are easier to extend
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#785] Simplify implementation of project <https://github.com/pymor/pymor/pull/785>`_
+- `[#619] Move all backend-agnostic assemble_lincomb logic to algorithms.lincomb <https://github.com/pymor/pymor/pull/619>`_
+
+
+Improvements to pyMOR's discretization toolkit
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#821] Enable to have parametric dirichlet in fv <https://github.com/pymor/pymor/pull/821>`_
+- `[#687] Discretizing robin boundary conditions on a RectGrid <https://github.com/pymor/pymor/pull/687>`_
+- `[#691] Remove 'order' arguments from CG operators <https://github.com/pymor/pymor/pull/691>`_
+- `[#760] [discretizers.cg] affine decomposition of robin operator and rhs functionals <https://github.com/pymor/pymor/pull/760>`_
+
+
+Infrastructure improvements
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#818] Tutorial setup <https://github.com/pymor/pymor/pull/818>`_
+- `[#793] Use meshio for Gmsh file parsing <https://github.com/pymor/pymor/pull/793>`_
+- `[#607] Replace sqlite caching <https://github.com/pymor/pymor/pull/607>`_
+
+
+
+Backward incompatible changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Dropped Python 3.5 support
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+- `[#553] increase minimal python version to 3.6 <https://github.com/pymor/pymor/pull/553>`_
+- `[#584] Simply implementation of RuleTable`_
```

Comment:
When running `make docs` there is an `Unknown target name` for this line. Maybe an URL is missing?